base/applications/network/telnet/src/tnerror.cpp

```diff
@@ -106,17 +163,21 @@ int printm(LPTSTR szModule, BOOL fSystem, DWORD dwMessageId, ...)
 				FORMAT_MESSAGE_FROM_STRING, szString, dwMessageId,
 				LANG_USER_DEFAULT, (LPTSTR)&pszMessage, 256, &Ellipsis);
 	}
+#endif
 
 	va_end(Ellipsis);
 
 	if (szModule)
 		FreeLibrary(hModule);
 
 	if (dwMessage) {
-
+#ifdef __REACTOS__
+		Result = wprintit(pszMessage);
+#else
 		Result = printit(pszMessage);
+#endif
 		LocalFree(pszMessage);
-	}
+    }
```

Comment:
```suggestion
	}
```
reverting the original indentation